sklearn/utils/tests/test_testing.py

```diff
@@ -211,26 +211,20 @@ def context_manager_no_user_multiple_warning():
     assert_warns(DeprecationWarning, context_manager_no_user_multiple_warning)
 
 
-# This class is inspired from numpy 1.7 with an alteration to check
-# the reset warning filters after calls to assert_warns.
-# This assert_warns behavior is specific to scikit-learn because
-# `clean_warning_registry()` is called internally by assert_warns
-# and clears all previous filters.
+# assert_warns will no longer clear all previous filters.
```

Comment:
Please remove this comment, it does add much value.

Reply:
Didn't quite understand, do you mean it is not true?